doc/source/notebooks/basics/GPLVM.ipynb

```diff
@@ -95,9 +93,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We construct a RBF kernel operating on the two-dimensional latent space. \n",
-    "The `ARD` parameter stands for 'Automatic Relevance Determination', which in practice means that\n",
-    "we learn a different lengthscale for each of the input dimensions, see [the kernels notebook](../advanced/kernels.ipynb)."
+    "We construct a Radial Basis Function (RBF) kernel operating on the two-dimensional latent space. \n",
```

Comment:
```suggestion
    "We construct a Squared Exponential (SE) kernel operating on the two-dimensional latent space. \n",
```
can also rename gpflow.kernels.RBF to gpflow.kernels.SquaredExponential in any code cells to make it consistent